profiles/base/package.use.mask

```diff
@@ -6,6 +6,32 @@
 # This file is only for generic masks. For arch-specific masks (i.e.
 # mask everywhere, unmask on arch/*) use arch/base.
 
+# hololeap <hololeap@protonmail.com> (2023-10-15)
+# Enabling these flags will create cycles in the dependency graph.
+# Maintainers may want to temporarily unmask these in order to run
+# tests for these packages, but it is necessary for each package to be
+# rebuilt with the flag disabled immediately after testing.
+# See: <https://github.com/gentoo-haskell/gentoo-haskell/pull/1375>
+dev-haskell/attoparsec test
+dev-haskell/base-orphans test
+dev-haskell/clock test
```

Comment:
Just to avoid confusion in the future, I wanted to point out that I'm adding a mask on `USE=test` for a couple dozen `dev-haskell` packages here. If my message doesn't explain why, I'm happy to clarify.

Reply:
Dependency cycles are bad for Haskell because if something gets ABI broken in the cycle, the whole cycle gets broken with no way out other than `--rage-clean`ing one or more packages.

Reply:
Adding `USE=test` to `package.use.mask` keeps these cycles from happening but allows for maintainers to run the tests periodically if they are aware of the dangers.